apiserver/facades/client/backups/backups.go

```diff
@@ -60,7 +60,7 @@ func NewAPI(backend Backend, resources facade.Resources, authorizer facade.Autho
 
 	// For now, backup operations are only permitted on the controller environment.
 	if !backend.IsController() {
-		return nil, errors.New("backups are not supported for hosted models")
+		return nil, errors.New("backups are only supported from the controller model")
```

Comment:
can we (on a new line) suggest how they switch to the controller model

Reply:
done